crates/bevy_reflect/derive/src/derive_data.rs

```diff
@@ -121,10 +121,9 @@ pub(crate) struct EnumVariant<'a> {
     /// The fields within this variant.
     pub fields: EnumVariantFields<'a>,
     /// The reflection-based attributes on the variant.
-    #[allow(dead_code)]
     pub attrs: FieldAttributes,
     /// The index of this variant within the enum.
-    #[allow(dead_code)]
+    #[expect(dead_code, reason = "TODO(MrGVSV): Why should this be kept?")]
```

Comment:
@MrGVSV can you help out with this one?

Reply:
Oh, I think they said in the previous PR that it could be removed, but I wasn't sure so of the implications so left the field there for the meantime.

Reply:
https://github.com/bevyengine/bevy/pull/15118#discussion_r1750326458

Reply:
Please remove it now then :)